--- ChangeLog	2026-05-03 18:20:04.959449289 -0400
+++ ChangeLog-spell-corrected	2026-05-03 18:15:22.039275870 -0400
@@ -20,7 +20,7 @@
 
 2026-04-20  rocky <rb@dustyfeet.com>
 
-	* README.rst, docker/Dockerfile-dev, pyproject.toml: Update partialy
+	* README.rst, docker/Dockerfile-dev, pyproject.toml: Update partially
 	for new 10.0.0 releases
 
 2026-04-20  rocky <rb@dustyfeet.com>
@@ -52,7 +52,7 @@
 	script/dmathics3-code-parse, script/{dmathics3-tokens =>
 	dmathics3-code-tokenize}, script/{dmathicsscript =>
 	dmathics3script}, script/{dmathicsscript.cmd =>
-	dmathics3script.cmd}, script/{dmathicsserver => dmathics3server}: 
+	dmathics3script.cmd}, script/{dmathicsserver => dmathics3server}:
 	Revise in preparation for next Mathics3 release (#21) * Massive changes for Mathics3 revision - Use Ubuntu 26:04 (llvm@18 now) - add mathics3-code-parser - correct entry to mathics3-server
 
 2026-03-22  rocky <rb@dustyfeet.com>
@@ -61,7 +61,7 @@
 
 2026-03-22  rocky <rb@dustyfeet.com>
 
-	* script/dmathics3.cmd: Add scirpt/dmathics3.cmd
+	* script/dmathics3.cmd: Add script/dmathics3.cmd
 
 2026-03-22  rocky <rb@dustyfeet.com>
 
@@ -71,7 +71,7 @@
 	=> mathics3_omnibus}/__init__.py, {mathics_omnibus =>
 	mathics3_omnibus}/version.py, pyproject.toml, script/dmathics.cmd,
 	script/dmathics3, script/{dmathicsdoccopy => dmathics3copy},
-	script/{dmathicsdoc => dmathics3doc}, script/dmathicsscript.cmd: 
+	script/{dmathicsdoc => dmathics3doc}, script/dmathicsscript.cmd:
 	Lots of little changes for 10.0 release...  Mathics -> Mathics3 lots of places.
 
 2026-03-06  R. Bernstein <rocky@users.noreply.github.com>
@@ -84,7 +84,7 @@
 
 2026-03-06  rocky <rb@dustyfeet.com>
 
-	* README.rst: Update README for isntall Also update SQLite db file.
+	* README.rst: Update README for install Also update SQLite db file.
 
 2025-10-04  rocky <rb@dustyfeet.com>
 
@@ -117,7 +117,7 @@
 
 2025-08-24  rocky <rb@dustyfeet.com>
 
-	* docker/Dockerfile, docker/Dockerfile-dev, docker/entrypoint.sh: 
+	* docker/Dockerfile, docker/Dockerfile-dev, docker/entrypoint.sh:
 	Use Python 3.13, git trepan3k, updated doc Use Python 3.13.  Use git trepan3k since it has passthrough
 	exceptions needed for Abort mathics3k-graph is now the package name
 
@@ -137,7 +137,7 @@
 	script/dmathics3, script/dmathics3-tokens, script/dmathicsdoc,
 	script/dmathicsdoccopy, script/dmathicsscript,
 	script/dmathicsserver: Update to Python 3.12, new API, new code and
-	Ubuntu 24.10 (#16) * Update to new API, code and Ubuntu 24.10 * mathicsdoc corretions for USER_HOME=/home/ubuntu * git pull Packages, set env var COLORFGBG... COLORFGBG is used to
+	Ubuntu 24.10 (#16) * Update to new API, code and Ubuntu 24.10 * mathicsdoc corrections for USER_HOME=/home/ubuntu * git pull Packages, set env var COLORFGBG... COLORFGBG is used to
 	make sure we have a compatible display style in trepan when
 	Breakpoint[] is used.  * Bump version number * pip install PyQt6 * Spell correct ChangeLog.  * Ignore mathics.pdf createdoutside * Go over pkginfo and NEWS.md * Go over README.rst
 
@@ -163,7 +163,7 @@
 
 2024-01-17  rocky <rb@dustyfeet.com>
 
-	* docker/Dockerfile, docker/Dockerfile-dev, docker/Dockerfile-git: 
+	* docker/Dockerfile, docker/Dockerfile-dev, docker/Dockerfile-git:
 	Use newer software versions in dev dockerfile
 
 2024-01-15  rocky <rb@dustyfeet.com>
@@ -250,7 +250,7 @@
 
 2022-07-17  rocky <rb@dustyfeet.com>
 
-	* docker/Dockerfile, docker/requirements.txt, script/dmathicsscript: 
+	* docker/Dockerfile, docker/requirements.txt, script/dmathicsscript:
 	Everything roughly working again.  * dmathicsscript * dmathicsserver * dmathicsdoc * dmathicsdoccopy However there are still some small rough edges
 
 2022-07-16  rocky <rb@dustyfeet.com>
@@ -338,7 +338,7 @@
 
 2021-09-12  rocky <rb@dustyfeet.com>
 
-	* : Add plots to Parital Diff Eqns 2.1
+	* : Add plots to Partial Diff Eqns 2.1
 
 2021-09-11  rocky <rb@dustyfeet.com>
 
@@ -387,7 +387,7 @@
 2021-08-08  rocky <rb@dustyfeet.com>
 
 	* script/dmathicsscript, script/dmathicsserver,
-	script/term-background.sh: Better persistance via setting Env
+	script/term-background.sh: Better persistence via setting Env
 	variables Better help for dmathicsscript and dmathicssserver Fix bug in term-background Revise symja example in mathics.sqlite
 
 2021-08-03  rocky <rb@dustyfeet.com>
@@ -423,7 +423,7 @@
 
 	* README.rst, __pkginfo__.py, docker/Dockerfile,
 	docker/Dockerfile-dev, docker/Dockerfile-dev+debug,
-	docker/entrypoint.sh, script/dmathicsdoc, script/dmathicsdoccopy: 
+	docker/entrypoint.sh, script/dmathicsdoc, script/dmathicsdoccopy:
 	Revise docker to allow PDF viewing and copy New scripts dmathicsdoc and dmathicsdoccopy perform the new
 	functions Document this in README.rst, and add these scripts in PyPI
 	packaging.  entrypoint.sh needs to multiplex dmathicsdoccpy and
@@ -734,7 +734,7 @@
 2021-02-20  rocky <rb@dustyfeet.com>
 
 	* .gitignore, README.rst, admin-tools/git2cl,
-	admin-tools/make-dist.sh, admin-tools/pyenv-versions, setup.py: 
+	admin-tools/make-dist.sh, admin-tools/pyenv-versions, setup.py:
 	Administrivia
 
 2021-02-20  R. Bernstein <rocky@users.noreply.github.com>
@@ -1999,10 +1999,9 @@
 	mathics/web/templates/base_html.html, mathics/web/urls.py,
 	mathics/web/views.py, pymathics/natlang/natlang.py, setup.py,
 	test/test_evaluation.py, test/test_file.py,
-	test/test_parser/test_tokeniser.py, tmp/README.rst, travis.py: 
+	test/test_parser/test_tokeniser.py, tmp/README.rst, travis.py:
 	Squash unrelated Docker items
 
 2020-12-25  rocky <rb@dustyfeet.com>
 
 	* Initial
-
